docker-compose.ha.yml

```diff
@@ -3,7 +3,7 @@ services:
   proxy:
     image: haproxy:1.7
     volumes:
-      - ./contrib/haproxy.cfg:/usr/local/etc/haproxy/haproxy.cfg:ro
+      - ./contrib/haproxy.cfg:/usr/local/etc/haproxy/haproxy.cfg:Z
     ports:
```

Comment:
According to https://docs.docker.com/storage/bind-mounts/#differences-between--v-and---mount-behavior you can have both `ro` and `Z` in a comma seperated list, not sure why only some files are `ro` though

Reply:
Oh, I was not aware of the option to provide a comma separated list. Let me give that a try.